certified-connectors/Haiilo/apiDefinition.swagger.json

```diff
@@ -16,7 +16,7 @@
     "/publicapi/channel": {
       "get": {
         "description": "Retrieve channel list for creating a new post",
-        "operationId": "SmarpRetrieveChannelList",
+        "operationId": "AdvocacyRetrieveChannelList",
```

Comment:
@thanh-nguyen The 'default' response at line 33. should not have schema definition. Schemas should be defined on expected responses only.